tracer/build/_build/Build.cs

```diff
@@ -334,6 +335,7 @@ void DeleteReparsePoints(string path)
         .DependsOn(CompileLinuxOrOsxIntegrationTests)
         .DependsOn(CompileLinuxDdDotnetIntegrationTests)
         .DependsOn(BuildRunnerTool)
+        .DependsOn(CopyNativeFilesForTests)
```

Comment:
We prob need this for `BuildOsxIntegrationTests` too I'm guessing? They don't run in CI, but people run them locally

Reply:
sure